app/models/allocated_lump_sum.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+
+from ..extensions import db
+from .mixins import TimestampMixin
+from .month_allocation import MonthAllocation
+
+
+class AllocatedLumpSum(db.Model, TimestampMixin):
+    """Allocated lump sum for a care month"""
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    # Relationships
+    care_month_allocation_id = db.Column(db.Integer, db.ForeignKey("month_allocation.id"), nullable=False)
+    care_month_allocation = db.relationship(
+        "MonthAllocation",
+        back_populates="lump_sums",
+        foreign_keys=[care_month_allocation_id],
+    )
+
+    # Calculated fields
+    amount_cents = db.Column(db.Integer, nullable=False)
+
+    # Provider info
+    provider_google_sheets_id = db.Column(db.String(64), nullable=False, index=True)
+
+    paid_at = db.Column(db.DateTime, nullable=True)
+    submitted_at = db.Column(db.DateTime, nullable=True)
+
+    @property
+    def is_paid(self) -> bool:
+        """Check if this lump sum has been paid."""
+        return self.paid_at is not None
+
+    def mark_as_paid(self):
+        """Mark this lump sum as paid."""
+        self.paid_at = db.func.current_timestamp()
```

Comment:
Can we use `datetime.now` so that we have access to the datetime object if we need it?
```suggestion
        self.paid_at = datetime.now(timezone.utc)
```

Reply:
Good call